ckanext/noanonaccess/plugin.py

```diff
@@ -24,6 +34,11 @@ def __call__(self, environ, start_response):
         elif 'repoze.who.identity' in environ or self._get_user_for_apikey(environ):
             # if logged in via browser cookies or API key, all pages accessible
             return self.app(environ,start_response)
+        elif dcat_access and (environ['PATH_INFO'].endswith(tuple(ext))
+                          or environ['PATH_INFO'].startswith(tuple(catalog_endpoints))):
```

Comment:
what's the point of defining list and  later doing `tuple(list)`

Why `tuple(ext)` here and not `ext = ('.jsonld','.xml','.ttl','.n3')` from the beginning? 

Same about `tuple(catalog_endpoints)`

Reply:
Or even, why `tuple` at all?